gating/gate/run

```diff
@@ -26,4 +26,25 @@
 # trigger.
 # RE_JOB_REPO_NAME: The project's GitHub repository which gets cloned in the
 # job. For example, for the repository found at
-# https://github.com/rcbops/rpc-gating, the name is rpc-gating.
\ No newline at end of file
+# https://github.com/rcbops/rpc-gating, the name is rpc-gating.
+
+set -eux
+set -o pipefail
+
+cd /opt/magnanimous-turbo-chainsaw/scripts/
+. setup.sh
```

Comment:
This would need to be `bash setup.sh`.

Reply:
Should have the same effect, no? Am I misunderstanding the capabilities of the all-powerful dot?

Reply:
Just realized that bash will execute in a new shell process, and there may be a reason why we want this (variables, etc).